gravitee-apim-plugin/gravitee-apim-plugin-endpoint/gravitee-apim-plugin-endpoint-http-proxy/src/main/resources/schemas/sharedConfiguration/schema-form.json

```diff
@@ -58,6 +58,17 @@
                 }
             }
         },
+        "propagateClientAcceptEncoding":{
+            "title":"Propagate client Accept-Encoding header",
+            "description": "Propagate Accept-Encoding can only be made if \"Enable compression\" is disabled.",
```

Comment:
What about: `Accept-Encoding can only be propagated if \"Enable compression\" is disabled.`